docs/eventing/sources/apiserversource/getting-started.md

```diff
@@ -186,14 +186,14 @@ command:
               --mode "Resource" \
               --resource "Event:v1" \
               --service-account <service-account> \
-              --sink <sink-name>
+              --sink <sink>
             ```
             Where:
 
             - `<apiserversource>` is the name of the source that you want to create.
             - `<namespace>` is the name of the namespace that you created in step 1 earlier.
             - `<service-account>` is the name of the ServiceAccount that you created in step 2 earlier.
-            - `<sink-name>` is the name of your sink, for example, `http://event-display.pingsource-example.svc.cluster.local`.
+            - `<sink>` is the sink you created in step 5 earlier, for example, Service url: `http://event-display.<namespace>.svc.cluster.local`
```

Comment:
We prefer to use more descriptive names like `sink-name` for variables in docs generally
@snneji any thoughts on this?